client/question/services/FeedbackGeneratorService.js

```diff
@@ -519,11 +631,12 @@ tie.factory('FeedbackGeneratorService', [
             var testCases = testSuites[j].getTestCases();
             for (var k = 0; k < testCases.length; k++) {
               var testCase = testCases[k];
+              var testCaseIndex = k;
+              var testSuiteId = testSuites[j]._id;
```

Comment:
Never reference a private member like this. Use getId() instead.

Reply:
Fixed.